docker/.env

```
@@ -183,7 +183,7 @@ UV_CUBE_PROXY_HOST=0.0.0.0
 UV_CUBE_PROXY_PORT=8900
 UV_CUBE_PROXY_SERVER_CERT=
 UV_CUBE_PROXY_SERVER_KEY=
-UV_CUBE_PROXY_TARGET_URL=http://cube-agent:8901
+UV_CUBE_AGENT_URL=http://cube-agent:8901
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The UV_CUBE_AGENT_URL key should go before the UV_CUBE_PROXY_HOST key

(UnorderedKey)